pkg/controller/splunkforwarder/splunkforwarder_controller.go

```diff
@@ -141,23 +141,20 @@ func (r *ReconcileSplunkForwarder) Reconcile(request reconcile.Request) (reconci
 	// Define a new ConfigMap object
 	// TODO(wshearn) - check instance.Spec.ClusterID, if it is empty look it up on the cluster.
 	configMaps := kube.GenerateConfigMaps(instance, request.NamespacedName, clusterid)
-	if instance.Spec.UseHeavyForwarder == true {
+	if instance.Spec.UseHeavyForwarder {
 		configMaps = append(configMaps, kube.GenerateInternalConfigMap(instance, request.NamespacedName))
 		configMaps = append(configMaps, kube.GenerateFilteringConfigMap(instance, request.NamespacedName))
 
 	}
 
-	// Define it outside the loop
-	cmFound := &corev1.ConfigMap{}
-
 	for _, configmap := range configMaps {
 		// Set SplunkForwarder instance as the owner and controller
 		if err := controllerutil.SetControllerReference(instance, configmap, r.scheme); err != nil {
 			return reconcile.Result{}, err
 		}
 
 		// Check if this ConfigMap already exists
-		cmFound = &corev1.ConfigMap{} // reset cmFound
+		cmFound := &corev1.ConfigMap{} // reset cmFound
```

Comment:
This comment no longer fits, it can be removed

Reply:
removed